docs/manual/getting_started/installation/core_cpp_common_build.rst

```diff
@@ -0,0 +1,27 @@
+.. The following text is included in installation.cpp.rst and core_cpp_common.rst (at the end of the files).
+
+If you do not require the examples, use ``-DBUILD_EXAMPLES=ON`` to omit them.
+
+To install |var-project-short| after a successful build:
+
+.. code-block:: bash
+
+    cmake --build . --target install
+
+The install step copies everything to:
+
+ -  ``<install-location>/lib``
+ -  ``<install-location>/bin``
+ -  ``<install-location>/include/ddsc``
+ -  ``<install-location>/share/CycloneDDS``
+
+.. note::
+    Depending on the installation location, you may need administrator privileges. 
+
+At this point, you are ready to use Cyclone DDS in your projects.
+
+.. note:: Build types
+
+    The default build type is a release build that includes debugging information (``RelWithDebInfo``).
+    This build is suitable for applications because of its high-performance and debugging capabilities.
```

Comment:
Maybe rephrase to:
"This build is useful in most generic cases, as it allows debugging to be done, while at the same time performing at a satisfactory level."

Reply:
I think these additions are as a result of spitting up the documentation and so it's not "new" content. However, this particular formulation was commented on in this previous pull-request: https://github.com/eclipse-cyclonedds/cyclonedds/pull/1413/files#r995468974 with the following suggestion:
> This build is suitable for applications because of its high-performance while still providing the ability for the application to be easily debugged.

My personal suggestion is as follows:
```suggestion
    This build is suitable for applications because it allows the resulting application to be more easily debugged while still maintaining high performance.
```

Reply:
Sorry, I didn't see your follow up question earlier:
I think you can find the available build types here:
https://github.com/eclipse-cyclonedds/cyclonedds/blob/7fb79b81456a203d02878aa4f9ef53f60fb202fa/CMakeLists.txt#L23